### PR TITLE
Adding ForceRunningNode Decorator

### PR DESCRIPTION
**Pull request by fmrico**
Hi Davide!!

While working in a PR for Navigation2, I found useful to develop this decorator node.

The idea is continuing ticking a node unless it returns FAILURE, even if it returns RUNNING. I think that this completes the series ForceFailureNode and ForceSuccessNode.

I hope it helps!!!

Best

Signed-off-by: Francisco Martin Rico <fmrico@gmail.com>

### CODE REVIEW

File: include/behaviortree_cpp_v3/decorators/force_running_node.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*  Copyright (C) 2018-2020 Davide Faconti, Eurecat -  All Rights Reserved
```
Comment (SteveMacenski) [inline include/behaviortree_cpp_v3/decorators/force_running_node.h:1]: I think you can add your name here Francisco :wink:

Comment (fmrico) [reply include/behaviortree_cpp_v3/decorators/force_running_node.h:1]: Thanks!! I was not sure 😊

**Comment by facontidavide** [comment]
Isn't it a just a RepeatUntilSuccesfull with no timeout?

**Comment by facontidavide** [comment]
I prefer a different name.

Since this reminds me RepeatUntilSuccesfull (with infinite retry). What about **KeepRunningUntilFailure** ?

**Comment by fmrico** [comment]
> Since this reminds me RepeatUntilSuccesfull (with infinite retry). What about KeepRunningUntilFailure ?

It's ok for me. changed ;)

**Comment by facontidavide** [comment]
cool, thanks

**Pull request merged**